src/InvestmentManager.sol

```diff
@@ -586,20 +586,14 @@ contract InvestmentManager is Auth, IInvestmentManager {
     }
 
     function _calculatePrice(address vault, uint128 assets, uint128 shares) internal view returns (uint256 price) {
```

Comment:
We don't need the separate anymore since we refactored the decrease/cancelation logic in a previous PR